docs/api/clients/typescript.md

```diff
@@ -46,7 +46,7 @@ By default, `ShapeStream` parses the following Postgres types into native JavaSc
 - `int8` is parsed into a JavaScript `BigInt`
 - `bool` is parsed into a JavaScript `Boolean`
 - `json` and `jsonb` are parsed into JavaScript values/arrays/objects using `JSON.parse`
-- Postgres Arrays are parsed into JavaScript arrays, e.g. `"{{1,2},{3,4}}"` is parsed into `[[1,2],[3,4]]`
+- Postgres Arrays are parsed into JavaScript arrays, e.g. `"&#123;{1,2},{3,4}}"` is parsed into `[[1,2],[3,4]]`
```

Comment:
huh what happened here?

Reply:
I explained it in the PR description above.

Prior to this change the build was failing with
```
9:59:58 AM: build error:
9:59:58 AM: [vite:vue] [plugin vite:vue] api/clients/typescript.md (32:75): Error parsing JavaScript expression: Unexpected token, expected "," (1:4)
9:59:58 AM: file: /opt/build/repo/docs/api/clients/typescript.md:32:75
9:59:58 AM: [vite:vue] [plugin vite:vue] api/clients/typescript.md (32:75): Error parsing JavaScript expression: Unexpected token, expected "," (1:4)
9:59:58 AM: file: /opt/build/repo/docs/api/clients/typescript.md:32:75
9:59:58 AM: SyntaxError: Error parsing JavaScript expression: Unexpected token, expected "," (1:4)
```

If you figure out there's a different root cause of the faliure, that'd be great!